tests/test_api.py

```diff
@@ -1453,9 +1461,9 @@ def test_closes(vfile):
 
     assert vfile._closed is True
     assert vfile.closed is True
-    raises(AttributeError, lambda: vfile.f)
-    raises(AttributeError, lambda: vfile._version_data)
-    raises(AttributeError, lambda: vfile._versions)
+    pytest.raises(AttributeError, lambda: vfile.f)
```

Comment:
Elsewhere `raises` is being converted to `with pytest.raises(...):` context manager, should the same approach be used here?